modules/library/src/main/scala/zio/elasticsearch/query/Queries.scala

```diff
@@ -121,6 +121,41 @@ private[elasticsearch] final case class Match[S, A: ElasticPrimitive](field: Str
   }
 }
 
+sealed trait HasParentQuery[S] extends ElasticQuery[S] with HasIgnoreUnmapped[HasParentQuery[S]] {
+  def withScore(value: Boolean): HasParentQuery[S]
+
+  def withScoreTrue: HasParentQuery[S] = withScore(true)
+
+  def withScoreFalse: HasParentQuery[S] = withScore(false)
```

Comment:
Maybe to extract this like rest of option params? Maybe it will occur in some other query?

Reply:
I would leave it for now as I haven't found any that have this field.